test/help/sexpr-wasm.txt

```diff
@@ -1,25 +1,25 @@
 ;;; EXE: %(sexpr-wasm)s
```

Comment:
Probably because they need to be updated in the python test runner too. It's ok, this change looks good to land. Thanks!